torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -890,6 +1107,13 @@ void ComputeConstant(Node* n, int opset_version) {
   }
 
   switch (n->kind()) {
+    case ::c10::onnx::Add:
```

Comment:
Do we need broadcasting for all ONNX ops that support bidirectional broadcasting? Like onnx And, BitShift, Equal, Greater, Less, Max, etc.

Reply:
There are more such ops. Please check.

Reply:
Will check all broadcasting ops. However, And, BitShift, Max is not in `interned_strings` so they are not used. Will not add them.

Reply:
Okay, anyways I think it'll be good to add them to interned_strings and this switch case too at some point.